File: src/collections/ArrayDemo.java
package collections;

import java.util.Arrays;

public class ArrayDemo {

    public static void main(String[] args) {

        int[] arrNumbers = {4, 12, 76, 48, 55, 23};

        System.out.println(Arrays.toString(arrNumbers));
        System.out.println("The 5th element of Array is " + arrNumbers[4]);

        for (int i = 0; i < arrNumbers.length; i++) {
            System.out.println("The index is " + i + ", the value is " + arrNumbers[i]);
        }
    }
}

File: src/collections/ArrayDemo.kt
package collections

fun main() {

    val arrNumbers: Array<Int> = arrayOf(4, 12, 76, 48, 55, 23)
    val arrNumbers2 = IntArray(5)

    println("The size of arrNumbers2 is ${arrNumbers2.size}.")

    for (num in arrNumbers) {
        println(num)
    }
}